docs/sdks/uid2-sdk-ref-android.md

```diff
@@ -81,9 +81,14 @@ The binary is published on Sonatype:
 
 ## Minimum Requirements
 
-Minimum requirements for this SDK are as follows:
+To consume the binary package of this SDK in your app:
 
-- Android Studio version: v2023.3.1+
+- Minimum target Android version: 4.4+ / API 19+ (SDK) 5.0+
+
+
+To run the development app as mentioned in the [Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side#client-side-integration-example) or build binary from source code, minimum requirements are as follows:
```

Comment:
To run the development app (see [Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side#client-side-integration-example)), or to build the binary from source code, the minimum requirements are as follows:

Reply:
fixed